schemas/v2.0.0.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+module.exports = {
+  version: '2.0.0',
+  baseUrl: 'https://prod-1-dc-api-oopeix3r.deepcrawl.com',
```

Comment:
don't you think this is going to be the same for everyone once they actually figure out the api?